app/src/ui/dialog/dialog.tsx

```diff
@@ -10,6 +10,11 @@ import { createUniqueId, releaseUniqueId } from '../lib/id-pool'
  */
 const dismissGracePeriodMs = 250
 
+/**
+ * Title bar height in pixels. Values taken from 'app/styles/_variables.scss'.
```

Comment:
This is great! Would you mind also adding a note in `_variables.scss` pointing to this file?